images/quay-upload/image-upload.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
```

Comment:
I would this image to `test`, as strictly speaking it is not an image use to operate pipeline-service, but used during development (on the other hand, if we put the image to update the gitops repo in this repo, it is part of operations and would be in `images`).

Reply:
As per the discussion on Gitlab, I moved it here.